nrf_wifi/hw_if/hal/inc/hal_interrupt.h

```diff
@@ -15,49 +11,39 @@
 #include "hal_api.h"
 
 /**
- * hal_rpu_irq_enable() - Enables interrupts from the RPU.
- * @hal_dev_ctx: Pointer to HAL context.
+ * @brief Enables interrupts from the RPU.
  *
- * This function enables the interrupts from the RPU.
+ * @param hal_dev_ctx Pointer to HAL context.
  *
- * Return: Status
- *		Pass : %NRF_WIFI_STATUS_SUCCESS
- *		Error: %NRF_WIFI_STATUS_FAIL
+ * @return Status
```

Comment:
Another minor comment - in some places in the documentation , it is `@return The status of the operation` and here above it is different. Should this be maintained same everywhere

Reply:
Yes, in this PR I focused on fixing doxygen warnings, not the content, I can take that up separately, there are plethora of things to fix as far as content is concerned.